torchvision/prototype/transforms/functional/_color.py

```diff
@@ -2,9 +2,29 @@
 from torchvision.prototype import features
 from torchvision.transforms import functional_pil as _FP, functional_tensor as _FT
 
-from ._meta import get_dimensions_image_tensor
+from ._meta import get_dimensions_image_tensor, get_num_channels_image_tensor
+
+
+def _blend(img1: torch.Tensor, img2: torch.Tensor, ratio: float) -> torch.Tensor:
+    ratio = float(ratio)
+    fp = img1.is_floating_point()
+    bound = 1.0 if fp else 255.0
+    output = img1.mul(ratio).add_(img2, alpha=(1.0 - ratio)).clamp_(0, bound)
+    return output if fp else output.to(img1.dtype)
+
+
+def adjust_brightness_image_tensor(img: torch.Tensor, brightness_factor: float) -> torch.Tensor:
```

Comment:
Please align the input names with all other `*_image_tensor` kernels.

```suggestion
def adjust_brightness_image_tensor(image: torch.Tensor, brightness_factor: float) -> torch.Tensor:
```